src/kOS/Binding/FlightControlManager.cs

```diff
@@ -319,16 +344,28 @@ public bool Enabled
                 {
                     SafeHouse.Logger.Log(string.Format("FlightCtrlParam: Enabled: {0} {1} => {2}", name, enabled, value));
 
-                    enabled = value;
-                    if (steeringManager != null)
+                    if (enabled != value)
                     {
-                        if (enabled) steeringManager.EnableControl(shared);
-                        else steeringManager.DisableControl();
-                        //steeringManager.Enabled = enabled;
-                    }
-                    if (RemoteTechHook.IsAvailable(control.Vessel.id))
-                    {
-                        HandleRemoteTechPilot();
+
+                        enabled = value;
+                        if (name == "steering")
+                        {
+                            SafeHouse.Logger.Log("FlightCtrlParam: toggle steering parameter");
+                            IFlightControlParameter param = kOSVesselModule.GetInstance(shared.Vessel).GetFlightControlParameter("steering");
+                            if (enabled)
+                            {
+                                param.EnableControl(shared);
+                            }
+                            else
+                            {
+                                param.DisableControl(shared);
+                            }
+                            return;
+                        }
+                        if (RemoteTechHook.IsAvailable(control.Vessel.id))
```

Comment:
We have code like this in couple of other places - a core kOS class depends on RT, which is an addon. If one day we decided to remove RT we'd have to edit quite a number of classes. It would be better for those classes to expose some kind of an interface that other components could plug in to. We do something like this with `UpdateHandler` - classes can subscribe to certain events.

I'm not going to ask you to change it now, we need a more general solution in the whole project. Just something to keep in mind.